### DISCUSSION TITLE
List of connectors?

### DISCUSSION THREAD

**Discussion by scottswigartpdx** [modelcontextprotocol/.github#107] [Q&A]
### Pre-submission Checklist

- [X] I have verified that this discussion would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Discussion Topic

Is there a list of existing connectors? Is there a connector for Sharepoint / Onedrive?

**Comment by jspahrsummers** [comment]
You can see a list of first-party integrations, and partial lists of official integrations and community integrations in https://github.com/modelcontextprotocol/servers. The README also contains links to some other resources that themselves list more integrations.
